File: drivers/fuchsia.rb
# frozen_string_literal: true

require "selenium-webdriver"

options = Selenium::WebDriver::Firefox::Options.new
driver = Selenium::WebDriver.for(:firefox, options: options)

begin
  driver.get "https://slurpcode.github.io/slurp/"
  driver.find_element(tag_name: "body").send_keys "matrix", :return
  sleep(30)
  driver.find_element(tag_name: "body").send_keys "jedi", :return
  sleep(30)
  driver.find_element(tag_name: "body").send_keys "midnight", :return
  sleep(15)
  driver.find_element(tag_name: "body").send_keys "king", :return
  sleep(15)
  # driver.find_element(tag_name: "body").send_keys "winter", :return
  # sleep(15)
ensure
  driver.quit
end

File: drivers/fuchsia.py
from time import sleep

from selenium import webdriver
from selenium.webdriver.common.by import By


driver = webdriver.Firefox()

driver.get("https://slurpcode.github.io/slurp/")
driver.find_element(By.TAG_NAME, "body").send_keys("matrix")
sleep(10)

driver.quit()
